packaging/makeself/makeself.sh

```diff
@@ -154,12 +118,21 @@ MS_Usage()
 }
 
 # Default settings
-if type gzip 2>&1 > /dev/null; then
+if type gzip >/dev/null 2>&1; then
     COMPRESS=gzip
+elif type compress >/dev/null 2>&1; then
+    COMPRESS=compress
```

Comment:
@Ferroin , please, take a look in this warning from `shellcheck`.

Reply:
I tested locally the static binary build process and also the normal installation and everything worked. I am only waiting for these warnings to be addressed before to approve.

Reply:
Should be resolved now.